Source/WebKit/UIProcess/API/glib/WebKitFindController.cpp

```diff
@@ -474,6 +484,8 @@ void webkit_find_controller_search_previous(WebKitFindController* findController
  * @find_options: a bitmask with the #WebKitFindOptions used in the search
  * @max_match_count: the maximum number of matches allowed in the search
  *
+ * Counts the number of matches for @search_text 
```

Comment:
Missing dot here, and trailing space:

```suggestion
 * Counts the number of matches for @search_text.
```